client/src/sendQuery.ts

```diff
@@ -31,14 +31,24 @@ export const buildSendQuery = async (input: {
   if (input.options.maxFeePerGas == undefined) {
     input.options.maxFeePerGas = await getMaxFeePerGas(input.axiom, input.options?.overrides);
   }
+
   const chainId = input.axiom.config.chainId.toString();
+  const axiomQueryAddress = input.options?.overrides?.queryAddress ?? getAxiomV2QueryAddress(chainId);
+  const abi = getAxiomV2Abi(AbiType.Query);
+
+  const publicClient = createPublicClient({
+    chain: viemChain(chainId, input.axiom.config.providerUri),
+    transport: http(input.axiom.config.providerUri),
+  });
+
+  const feeDataExtended = await calculateFeeDataExtended(chainId, publicClient, input.options);
+  const payment = await calculatePayment(chainId, publicClient, feeDataExtended);
```

Comment:
`payment` looks like it is not needed in this function anymore? 
If it is actually not used anywhere, then we can delete `calculatePayment` to avoid future confusion